tests/src/lib.rs

```diff
@@ -0,0 +1,72 @@
+use anyhow::Result;
+use common::{random_payload, send_get, send_post, OrderCreatedResponse, OrderStatusResponse};
+use std::time;
+mod common;
+
+const RETRY_TIMES: u32 = 5;
```

Comment:
nit: RETRY_TIMES -> MAX_RETRY_TIMES?

Reply:
of course, or aligning with use in our products we could take MAX_RETRY_ATTEMPTS?